WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -112,11 +112,13 @@ <h1>Privacy <span class="green-text d-block">by default</span></h1>
                                 </p>
                             </div>
                         </div>
+                        <!--
```

Comment:
Why was this commented out? Will this image be replaced?

Reply:
That image was ugly, and without it, the website looks much better.

Reply:
Okay, then we should just remove it.

Reply:
You're right! Removed it.